doc/user/INSTALLATION.md

```diff
@@ -1,18 +1,19 @@
 # Installation
 
-## Current release packages (1.3.1)
+## Current release packages (2.0.0)
 
 | Platform | Files |
 | -------- | ----- |
-| Windows Installer | [f3d-1.3.1-win64.exe](https://github.com/f3d-app/f3d/releases/download/v1.3.1/f3d-1.3.1-win64.exe) |
-| Windows (Portable) | [f3d-1.3.1-win64.zip](https://github.com/f3d-app/f3d/releases/download/v1.3.1/f3d-1.3.1-win64.zip) |
-| MacOS | [f3d-1.3.1-Darwin.dmg](https://github.com/f3d-app/f3d/releases/download/v1.3.1/f3d-1.3.1-Darwin.dmg) |
-| Debian-based Package | [f3d-1.3.1-Linux.deb](https://github.com/f3d-app/f3d/releases/download/v1.3.1/f3d-1.3.1-Linux.deb) |
-| Linux (Portable) | [f3d-1.3.1-Linux.tar.xz](https://github.com/f3d-app/f3d/releases/download/v1.3.1/f3d-1.3.1-Linux.tar.xz) |
+| Windows Installer | [F3D-2.0.0-Windows.exe](https://github.com/f3d-app/f3d/releases/download/v2.0.0/F3D-2.0.0-Windows-x86_64-raytracing.exe) |
```

Comment:
Should we add the no-raytracing alternatives, and also add a column with the size of the package?

Reply:
I thoughts about that but it would make this as complex as the releases pages so I decided not to.

I think we need the javascript page more than ever. I have a plan for that :p